chapters/substitution.adoc

```diff
@@ -2,13 +2,25 @@
 .Definition
 Robots are substituted by the <<Robot Handler, robot handler>> of the respective team. No other team member is allowed to take robots out or put robots in.
 
-. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
-.. Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
-.. Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
+. Robot substitution during game play:
+.. The <<Robot Handler, robot handler>> should prefer to use long sleeves and colors that won't interfere with the vision system.
+.. A robot can only be substituted while obeying the following specifications:
+... Is at least on top of any touch line, if possible the robot can also be placed beyond the field boundaries
```

Comment:
Suggestion: Is at least partially inside the field margin

but double check if the field margin is defined as such. Maybe add it into the field drawings?